docs/remote/containers-advanced.md

```diff
@@ -82,68 +120,74 @@ If you've already built the container and connected to it, run **Remote-Containe
 
-Since this sample [runs VS Code as non-root user](#adding-a-non-root-user-to-your-dev-container) in the container called "node", a `postCreateCommand` is included to make sure the user can access the `node_modules` folder.
+    ```json
+    "workspaceMount": "src=${localWorkspaceFolder},dst=/workspace,type=bind,consistency=cached",
+    "workspaceFolder": "/workspace",
+    "runArgs": [
+        "-u", "node",
+        "-v", "try-node-node_modules:/workspace/node_modules"
+    ],
+    "postCreateCommand": "sudo chown node:node node_modules"
+    ```
 
-```json
-"workspaceMount": "src=${localWorkspaceFolder},dst=/workspace,type=bind,consistency=cached",
-"workspaceFolder": "/workspace",
-"runArgs": [
-    "-u", "node",
-    "-v", "try-node-node_modules:/workspace/node_modules"
-],
-"postCreateCommand": "sudo chown node:node node_modules"
-```
+    This second step is not required if you will be running in the container as `root`.
 
 If you've already built the container and connected to it, run **Remote-Containers: Rebuild Container** from the Command Palette (`kbstyle(F1)`) to pick up the change. Otherwise run **Remote-Containers: Open Folder in Container...** to connect to the container.
 
 **Docker Compose**:
-In this case, you will add the named volume to your Docker Compose file (or an [extended one](/docs/remote/containers.md#extending-your-docker-compose-file-for-development)) for the appropriate service(s). For example:
 
-```yaml
-version: '3'
-services:
-  your-service-name-here:
-    volumes:
-       # Or wherever you've mounted your source code
-      - .:/workspace
-      - try-node-node_modules:/workspace/node_modules
+The steps are identical for Docker Compose, but the volume mount configuration is simply placed in a different file.
 
-volumes:
-  - try-node-node_modules:
-```
+1. In your Docker Compose file (or an [extended one](/docs/remote/containers.md#extending-your-docker-compose-file-for-development)), add a named local volume mount to the `node_modules` sub-folder for the appropriate service(s). For example:
 
-You'll also want to be sure the `workspaceFolder` property in `devcontainer.json` matches the place your actual source code mounted:
+    ```yaml
+    version: '3'
+    services:
+      your-service-name-here:
+        volumes:
+          # Or wherever you've mounted your source code
+          - .:/workspace
+          - try-node-node_modules:/workspace/node_modules
+        # ...
 
-```json
-"workspaceFolder": "/workspace"
-```
+    volumes:
+      - try-node-node_modules:
+    ```
 
-If you're running in the container with a [user other than root](#adding-a-non-root-user-to-your-dev-container), add a postCreateCommand to update the owner of the folder you mount since it may have been mounted as root.
+2. Next, be sure the `workspaceFolder` property in `devcontainer.json` matches the place your actual source code mounted:
```

Comment:
"is" mounted?

Reply:
Yep - made this change in vnext.  Thanks.